src/libraries/System.Private.CoreLib/src/System/Runtime/JitInfo.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime
+{
+    public static partial class JitInfo
+    {
+        /// <summary>
+        /// Get the amount of time the JIT Compiler has spent compiling methods. If <paramref name="currentThread"/> is true,
+        /// then this value is scoped to the current thread, otherwise, this is a global value.
+        /// </summary>
+        /// <param name="currentThread">Whether the returned value should be specific to the current thread. Default: false</param>
+        /// <returns>The amount of time the JIT Compiler has spent compiling methods.</returns>
+        public static TimeSpan GetCompilationTime(bool currentThread = false)
+        {
+            return TimeSpan.FromTicks(GetCompilationTimeInTicks(currentThread));
```

Comment:
Is this implementation correct for CoreCLR? `GetCompilationTimeInTicks` uses QPC ticks, where `Stopwatch.Frequency` ticks represent one second (e.g., on my machine, that value is 10,000,000). However, `TimeSpan.FromTicks` uses FILETIME ticks, where exactly 10,000 ticks represent one second.

Reply:
This should be something like
```
RawTicks * TickFrequency
where TickFrequency is TicksPerSecond / Frequency
```
https://source.dot.net/#System.Private.CoreLib/Stopwatch.cs,134